docs/src/free_associative_algebra.md

```diff
@@ -218,3 +218,63 @@ julia> collect(exponent_words(3*b*a*c - b + c + 2))
  []
 ```
 
+### Groebner bases
+
+The function `groebner_basis` provides the computation of a groebner basis of an ideal, given a set of 
+generators of that ideal.
+Since such a groebner basis is not necessarily finite, one can additionally pass a `reduction_bound`
+to the function, to only compute a partial groebner basis.
+
+**Examples**
+
+```jldoctest; setup = :(using AbstractAlgebra.Generic, AbstractAlgebra)
```

Comment:
These doctests fails (you can see details at https://github.com/Nemocas/AbstractAlgebra.jl/pull/1292/files)

It's also strange that you need to import `AbstractAlgebra.Generic`; I'd rather avoid that and insert explicit `Generic.`, as that is closer to what a user inputing code would experience